### PR TITLE
Add changelog entries for recently added features.

### PR DESCRIPTION
**Pull request by neonichu**

### CODE REVIEW

**Comment by neonichu** [comment]
@swift-ci please smoke test

**Review by tomerd — APPROVED** [review]
<3

**Comment by neonichu** [comment]
@swift-ci please smoke test windows

**Comment by neonichu** [comment]
@swift-ci please smoke test macOS

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,28 @@ Swift Next
 
   In packages that specify resources using a future tools version, the generated resource bundle accessor will import `Foundation.Bundle` for its own implementation only. _Clients_ of such packages therefore no longer silently import `Foundation`, preventing inadvertent use of Foundation extensions to standard library APIs, which helps to avoid unexpected code size increases.
 
+* [#6067]
+
+  Basic support for a new `.embed` resource rule which will allow embedding the contents of the resource into the executable code by generating a byte array, e.g.
```
Comment (MaxDesiatov) [inline CHANGELOG.md:12]: wording nit
```suggestion
  Basic support for a new `.embed` resource rule which allows embedding the contents of the resource into the executable code by generating a byte array, e.g.
```

**Comment by neonichu** [comment]
@swift-ci please smoke test

**Comment by neonichu** [comment]
@swift-ci please smoke test macOS

**Review by MaxDesiatov — APPROVED** [review]

**Pull request merged**